v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Create Security Certificates
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
+toc: true
+---
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires a Certificate Authority (CA) certificate as well as keys and certificates for nodes, clients, and (optionally) the Admin UI. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, use [`openssl` commands](https://wiki.openssl.org/index.php/), or use a custom CA (for example, a public CA or your organizational CA).
+
+<div class="filters filters-big clearfix">
+  <a href="create-security-certificates.html"><button style="width:28%" class="filter-button">Use cockroach cert</button>
+  <a href="create-security-certificates-openssl.html"><button style="width:28%" class="filter-button">Use Openssl</button></a>
+  <button style="width:28%" class="filter-button current">Use <strong>custom CA</strong></button></a>
+</div>
+
+This document discusses the following advanced use cases for using security certificates with CockroachDB:
```

Comment:
Should there be a "simple" example for custom CAs, discussing the fields that must be set for the minimum configuration?  We should also mention that the three advanced use cases can be combined.